### PR TITLE
I taught you better than this @nathan-chin

### PR DESCRIPTION
**Pull request by lalaloosay**

### CODE REVIEW

**Comment by copperstick6** [comment]
?

**Comment by lalaloosay** [comment]
https://owl.english.purdue.edu/owl/resource/566/01/

File: index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
         </div>
         <section id="intro">
             <div class="wrap">
-                <h1>We're Freetail Hackers</h1>
+                <h1>We're Freetail Hackers.</h1>
```
Comment (copperstick6) [inline index.html:22]: ?

**Review by copperstick6 — CHANGES_REQUESTED** [review]

**Comment by copperstick6** [comment]
@lalaloosay [https://www.huffingtonpost.com/2015/03/19/twitter-grammar-no-punctuation_n_6785766.html](https://www.huffingtonpost.com/2015/03/19/twitter-grammar-no-punctuation_n_6785766.html)

**Comment by copperstick6** [comment]
Formal Protest

**Comment by rohitdatta** [comment]
I am filing an amicus brief in support of this pull request

**Comment by copperstick6** [comment]
Informal Protest

**Pull request merged**